content/applications/finance/accounting/customer_invoices/deferred_revenues.rst

```diff
@@ -1,195 +1,206 @@
 =================
-Deferred revenues
+Deferred Revenues
 =================
 
-**Deferred revenues**, or **unearned revenue**, are payments made in advance by customers for
-products yet to deliver or services yet to render.
+**Deferred revenues**, or **unearned revenues**, are invoices addressed to customers
+for goods yet to deliver or services yet to render.
 
-Such payments are a **liability** for the company that receives them since it still owes its
-customers these products or services. The company cannot report them on the current **Profit and
-Loss statement**, or *Income Statement*, since the payments will be effectively earned in the future.
+The company cannot report them on the current **Profit and Loss statement**, or *Income Statement*,
+since the goods and services will be effectively delivered/rendered in the future.
 
-These future revenues must be deferred on the company's balance sheet until the moment in time they
-can be **recognized**, at once or over a defined period, on the Profit and Loss statement.
+These future revenues must be deferred on the company's balance sheet among the current liabilities
+until they can be **recognized**, at once or over a defined period, on the Profit and Loss statement.
 
-For example, let's say we sell a five-year extended warranty for $ 350. We already receive the money
-now but haven't earned it yet. Therefore, we post this new income in a deferred revenue account and
-decide to recognize it on a yearly basis. Each year, for the next 5 years, $ 70 will be recognized
-as revenue.
+For example, let's say we sell a software license of $ 1200 for 1 year.
+We immediately invoice it to the customer but can't considered having it earned it yet.
+Therefore, we post this new revenue in a deferred revenue account and
+decide to recognize it on a monthly basis. Each month, for the next 12 months, $ 100 will be
+recognized as an revenue.
 
 Odoo Accounting handles deferred revenues by spreading them in multiple entries that are
-automatically created in *draft mode* and then posted periodically.
+posted periodically.
 
-.. note::
-   The server checks once a day if an entry must be posted. It might then take up to 24 hours before
-   you see a change from *draft* to *posted*.
```

Comment:
This is still true depending on the settings, sure we want to remove? (if not, I think that "Odoo" is better than "The server")

Reply:
We decided to remove it with Benj. This is an explanation of the auto-post mechanism not deferred